stripe/src/main/java/com/stripe/android/view/CardInputWidget.java

```diff
@@ -743,6 +777,18 @@ protected void onLayout(boolean changed, int l, int t, int r, int b) {
         }
     }
 
+    private static boolean isCvcMaximalLength(boolean isAmex, @Nullable String cvcText) {
```

Comment:
can we make this more generic, and take the card brand as a parameter, rather than an `isAmex` boolean?  currently, we're always calling `Card.AMERICAN_EXPRESS.equals(brand)` before calling this function, but that could just live here.